examples/generate.sh

```diff
@@ -94,14 +97,26 @@ ENVSUBST="${SOURCE_DIR}/envsubst-go"
 curl --fail -Ss -L -o "${ENVSUBST}" https://github.com/a8m/envsubst/releases/download/v1.2.0/envsubst-"$(uname -s)"-"$(uname -m)"
 chmod +x "$ENVSUBST"
 
+SRC_DIR="${SOURCE_DIR}"
+REORDER_TYPE="--reorder=legacy"
```

Comment:
Why is this needed?

Reply:
Mainly `--reorder=none` is required for clusterclass example. By default kustomization will use `reorder=legacy` and will reorde definitions in the outcome file alphabetically. Now, if the definitions are ordered and ClusterClass will be in the outcome file **above** the Metal3 definitions (e.g. `Metal3DataTemplate`), during deletion with `make delete-examples-clusterclass` the operation will fail as k8s will first delete ClusterClass and associated Metal3 objects and will try to delete Metal3 objects once again. At least that's what's happening to me, hence `reored=none` was added for ClusterClass example and `reorder=legacy` was added for non-ClusterClass example generation.

However, I've deleted this `reorder=legacy` and now only `reorder=none` will be used and only for ClusterClass.